ui_framework/doc_site/src/views/table/controlled_table_with_no_items.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiToolBarSection,
+  KuiButton,
+  KuiButtonIcon,
+  KuiToolBarText,
+  KuiControlledTable,
+  KuiPager,
+  KuiToolBarFooterSection,
+  KuiToolBarFooter,
+  KuiTableInfo,
+  KuiEmptyTablePromptPanel,
+} from '../../../../components';
+
+export class ControlledTableWithNoItems extends React.Component {
+  getPager() {
+    return (
+      <KuiPager
+        startNumber={ 1 }
+        hasNextPage={ true }
+        hasPreviousPage={ false }
+        endNumber={ 10 }
+        totalItems={ 100 }
+        onNextPage={ () => {} }
+        onPreviousPage={ () => {} }
+      />
+    );
+  }
+
+  render() {
+    return (
+      <KuiControlledTable>
+        <KuiToolBar>
+          <KuiToolBarSearchBox onFilter={ () => {} } />
+
+          <KuiToolBarSection>
+            <KuiButton buttonType="primary">
+              Add
+            </KuiButton>
+
+            <KuiButton
+              buttonType="basic"
+              icon={<KuiButtonIcon type="settings" />} />
```

Comment:
Can we move the closing `/>` to its own line and line it up with the opening `<KuiButton`?